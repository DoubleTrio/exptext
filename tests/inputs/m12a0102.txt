def 0 {
    sound_Stop();
    back_SetGround(LEVEL_G01P01A);
    supervision_StationCommon(0);
    supervision_LoadStation(LEVEL_G01P01A, 'UM12');
    supervision_Station(2);
    supervision_Acting(2);
    with (actor ACTOR_NPC_RURIRI) {
        SetPositionInitial();
    }
    with (actor ACTOR_NPC_MARIRU) {
        SetPositionInitial();
    }
    with (actor ACTOR_NPC_RURIRI) {
        SetAnimation(2);
    }
    with (actor ACTOR_NPC_MARIRU) {
        SetAnimation(2);
    }
    with (performer 0) {
        camera_SetMyself();
    }
    bgm2_PlayFadeIn(BGM_OCEAN3, 60, 190);
    screen_FadeIn(1, 30);
    Wait(30);
    se_Play(8978);
    with (actor ACTOR_NPC_RURIRI) {
        SetEffect(EFFECT_EXCLAMATION_MARK, 3);
    }
    with (actor ACTOR_NPC_MARIRU) {
        SetEffect(EFFECT_EXCLAMATION_MARK, 3);
    }
    supervision_Acting(0);
    Wait(5);
    with (actor ACTOR_PLAYER) {
        MovePositionOffset(1, 0, 12);
    }
    with (actor ACTOR_PLAYER) {
        Move2PositionMark(1, Position<'m0', 32, 22>);
    }
    supervision_Acting(1);
    Wait(1);
    with (actor ACTOR_ATTENDANT1) {
        MovePositionOffset(1, 0, 12);
    }
    with (actor ACTOR_PLAYER) {
        Turn2Direction(4, 10, DIR_DOWN);
    }
    with (actor ACTOR_ATTENDANT1) {
        Move2PositionMark(1, Position<'m1', 28, 22>);
    }
    with (actor ACTOR_ATTENDANT1) {
        Turn2Direction(4, 10, DIR_DOWN);
    }
    WaitExecuteLives(ACTOR_ATTENDANT1);
    message_SetFace(ACTOR_NPC_RURIRI, FACE_NORMAL, FACE_POS_BOTTOM_L_FACEINW);
    message_Talk({
        english=" [partner] and [hero]!\nHi there!",
    });
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    message_SetFace(ACTOR_ATTENDANT1, FACE_NORMAL, FACE_POS_TOP_L_CENTER);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" Huh? It\'s you little guys...",
            }
        case 2:
            {
                english=" Oh! It\'s you little guys...",
            }
        default:
            {
                english=" Hey! It\'s you little guys...",
            }
    }
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" We heard we have visitors.\nAre you our visitors?",
            }
        case 2:
            {
                english=" We heard we have visitors.\nAre you our visitors?",
            }
        default:
            {
                english=" We heard we have visitors. Are\nyou maybe those visitors?",
            }
    }
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_NPC_MARIRU) {
        Turn2DirectionLives(4, 10, ACTOR_ATTENDANT1);
    }
    WaitExecuteLives(ACTOR_NPC_MARIRU);
    message_SetFace(ACTOR_NPC_MARIRU, FACE_NORMAL, FACE_POS_BOTTOM_R_FACEINW);
    message_Talk({
        english=" Yep![K] We wanted to ask\nTeam [team:] for help.",
    });
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_ATTENDANT1) {
        Turn2DirectionLives(4, 10, ACTOR_NPC_MARIRU);
    }
    WaitExecuteLives(ACTOR_ATTENDANT1);
    message_SetFace(ACTOR_ATTENDANT1, FACE_NORMAL, FACE_POS_TOP_L_CENTER);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" Help?",
            }
        case 2:
            {
                english=" Help?",
            }
        default:
            {
                english=" Help?",
            }
    }
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    message_SetFace(ACTOR_NPC_MARIRU, FACE_NORMAL, FACE_POS_BOTTOM_R_FACEINW);
    message_Talk({
        english=" Yep!",
    });
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_NPC_RURIRI) {
        ExecuteCommon(CORO_JUMP_HAPPY_FUNC_SERIES, 0);
    }
    WaitExecuteLives(ACTOR_NPC_RURIRI);
    message_SetFace(ACTOR_NPC_RURIRI, FACE_NORMAL, FACE_POS_STANDARD);
    message_Talk({
        english=" We need your help finding a\n[CS:Z]Water Float[CR].",
    });
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_ATTENDANT1) {
        Turn2DirectionLives(4, 10, ACTOR_NPC_RURIRI);
    }
    WaitExecuteLives(ACTOR_ATTENDANT1);
    se_Play(8962);
    with (actor ACTOR_ATTENDANT1) {
        SetEffect(EFFECT_QUESTION_MARK, 3);
    }
    with (actor ACTOR_ATTENDANT1) {
        WaitEffect();
    }
    WaitExecuteLives(ACTOR_ATTENDANT1);
    message_SetFace(ACTOR_ATTENDANT1, FACE_NORMAL, FACE_POS_TOP_L_CENTER);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" A [CS:Z]Water Float[CR]?",
            }
        case 2:
            {
                english=" A [CS:Z]Water Float[CR]?",
            }
        default:
            {
                english=" A [CS:Z]Water Float[CR]?",
            }
    }
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" You\'ve been searching for your\n[CS:Z]Water Float[CR], right?",
            }
        case 2:
            {
                english=" Weren\'t you looking for that\nearlier?[K] A [CS:Z]Water Float[CR]?",
            }
        default:
            {
                english=" Isn\'t that what you were looking\nfor earlier?[K] A [CS:Z]Water Float[CR]?",
            }
    }
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_ATTENDANT1) {
        Turn2DirectionLives(4, 10, ACTOR_NPC_MARIRU);
    }
    message_SetFace(ACTOR_NPC_MARIRU, FACE_SIGH, FACE_POS_BOTTOM_R_FACEINW);
    message_Talk({
        english=" Yep! Someone said one was lying\non the beach. We went there looking for it!",
    });
    message_Talk({
        english=" We didn\'t find it. Instead, this is\nwhat we found.",
    });
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_NPC_MARIRU) {
        MovePositionOffset(32896, -8, -8);
    }
    with (actor ACTOR_PLAYER) {
        Turn2Direction(4, 10, DIR_DOWNLEFT);
    }
    WaitExecuteLives(ACTOR_NPC_MARIRU);
    Wait(15);
    se_Play(5124);
    supervision_Acting(3);
    Wait(15);
    with (actor ACTOR_NPC_MARIRU) {
        SetAnimation(5);
    }
    with (actor ACTOR_NPC_MARIRU) {
        SlidePositionOffset(32896, 8, 8);
    }
    WaitExecuteLives(ACTOR_NPC_MARIRU);
    with (actor ACTOR_NPC_MARIRU) {
        SetAnimation(2);
    }
    Wait(15);
    se_Play(8962);
    with (actor ACTOR_ATTENDANT1) {
        SetEffect(EFFECT_QUESTION_MARK, 3);
    }
    with (actor ACTOR_ATTENDANT1) {
        WaitEffect();
    }
    WaitExecuteLives(ACTOR_ATTENDANT1);
    message_SetFace(ACTOR_ATTENDANT1, FACE_WORRIED, FACE_POS_TOP_L_CENTER);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" This scrap of paper?",
            }
        case 2:
            {
                english=" This scrap of paper?",
            }
        default:
            {
                english=" This scrap of paper?",
            }
    }
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" There\'s writing on it.",
            }
        case 2:
            {
                english=" There\'s writing on it.",
            }
        default:
            {
                english=" There\'s writing on it.",
            }
    }
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (object OBJECT_G01P01A2_94) {
        SetOutputAttribute(2);
    }
    with (actor ACTOR_ATTENDANT1) {
        MovePositionOffset(32896, 8, 8);
    }
    WaitExecuteLives(ACTOR_ATTENDANT1);
    Wait(15);
    se_Play(5124);
    supervision_RemoveActing(3);
    Wait(15);
    with (actor ACTOR_ATTENDANT1) {
        SetAnimation(5);
    }
    with (actor ACTOR_ATTENDANT1) {
        SlidePositionOffset(32896, -8, -8);
    }
    WaitExecuteLives(ACTOR_ATTENDANT1);
    with (actor ACTOR_ATTENDANT1) {
        SetAnimation(2);
    }
    Wait(30);
    bgm2_FadeOut(60);
    with (actor ACTOR_PLAYER) {
        Turn2DirectionLives(4, 10, ACTOR_ATTENDANT1);
    }
    message_Mail({
        english="[CN][partner] read the message\n[CN]on the scrap of paper.",
    });
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    message_SetFace(ACTOR_ATTENDANT1, FACE_NORMAL, FACE_POS_TOP_L_CENTER);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" Let\'s see. What does it say?[K]\n\"The [CS:Z]Water Float[CR] from the beach...\"",
            }
        case 2:
            {
                english=" Let\'s see. What does it say?[K]\n\"The [CS:Z]Water Float[CR] from the beach...\"",
            }
        default:
            {
                english=" Let\'s see. What does it say?[K]\n\"The [CS:Z]Water Float[CR] from the beach...\"",
            }
    }
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" \"...is now in our possession.\"",
            }
        case 2:
            {
                english=" \"...is now in our possession.\"",
            }
        default:
            {
                english=" \"...is now in our possession.\"",
            }
    }
    message_KeyWait();
    bgm_PlayFadeIn(BGM_GROWING_ANXIETY, 0, 256);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" \"Just try to take it from us!\nWe\'ll wait in the deepest part of [CS:P]Amp Plains[CR].\"",
            }
        case 2:
            {
                english=" \"Just try to take it from us!\nWe\'ll wait in the deepest part of [CS:P]Amp Plains[CR].\"",
            }
        default:
            {
                english=" \"Just try to take it from us!\nWe\'ll wait in the deepest part of [CS:P]Amp Plains[CR].\"",
            }
    }
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" \"But knowing how weak and puny\nyou are...\"",
            }
        case 2:
            {
                english=" \"But knowing how weak and puny\nyou are...\"",
            }
        default:
            {
                english=" \"But knowing how weak and puny\nyou are...\"",
            }
    }
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" \"...I bet you can\'t even reach us!\nChaw-haw-haw!\"",
            }
        case 2:
            {
                english=" \"...I bet you can\'t even reach us!\nChaw-haw-haw!\"",
            }
        default:
            {
                english=" \"...I bet you can\'t even reach us!\nChaw-haw-haw!\"",
            }
    }
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" \"Can\'t handle it? Go cry to your\nbig-shot friends! Chaw-haw-haw!\"[K]\nHey! This note! It\'s a...",
            }
        case 2:
            {
                english=" \"Can\'t handle it? Go cry to your\nbig-shot friends! Chaw-haw-haw!\"[K]\nHey! This note! It\'s a...",
            }
        default:
            {
                english=" \"Can\'t handle it? Go cry to your\nbig-shot friends! Chaw-haw-haw!\"[K]\nHey! This note! It\'s a...",
            }
    }
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    se_Play(8968);
    with (actor ACTOR_ATTENDANT1) {
        SetEffect(EFFECT_SHOCKED_MIRRORED, 3);
    }
    with (actor ACTOR_ATTENDANT1) {
        WaitEffect();
    }
    WaitExecuteLives(ACTOR_ATTENDANT1);
    message_SetFace(ACTOR_ATTENDANT1, FACE_SURPRISED, FACE_POS_TOP_L_CENTER);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" It\'s a ransom note! Isn\'t it?!",
            }
        case 2:
            {
                english=" It\'s a ransom note! Isn\'t it?!",
            }
        default:
            {
                english=" It\'s a ransom note! Isn\'t it?!",
            }
    }
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_PLAYER) {
        Turn2DirectionLives(4, 10, ACTOR_NPC_MARIRU);
    }
    with (actor ACTOR_ATTENDANT1) {
        ExecuteCommon(CORO_JUMP_ANGRY_FUNC_SERIES, 0);
    }
    WaitExecuteLives(ACTOR_ATTENDANT1);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" You little guys shouldn\'t even\nthink about going!",
            }
        case 2:
            {
                english=" You little guys! Promise me you\nwon\'t go yourself.",
            }
        default:
            {
                english=" You little guys! Please don\'t even\nthink about going.",
            }
    }
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" It could be a trap.",
            }
        case 2:
            {
                english=" I suspect a trap.",
            }
        default:
            {
                english=" It\'s probably a trap!",
            }
    }
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    bgm_FadeOut(120);
    message_SetFace(ACTOR_NPC_MARIRU, FACE_SIGH, FACE_POS_BOTTOM_R_FACEINW);
    message_Talk({
        english=" But the [CS:Z]Water Float[CR] is very\nimportant to us.",
    });
    message_Talk({
        english=" We really want to get it back.",
    });
    message_Talk({
        english=" But I can\'t take [CS:N]Azurill[CR] to a\ndangerous place.",
    });
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    bgm2_PlayFadeIn(BGM_OCEAN3, 90, 190);
    with (actor ACTOR_NPC_RURIRI) {
        ExecuteCommon(CORO_JUMP_HAPPY_FUNC_SERIES, 0);
    }
    WaitExecuteLives(ACTOR_NPC_RURIRI);
    message_SetFace(ACTOR_NPC_RURIRI, FACE_NORMAL, FACE_POS_STANDARD);
    message_Talk({
        english=" I said I would go too!",
    });
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_NPC_MARIRU) {
        Turn2DirectionLives(4, 10, ACTOR_NPC_RURIRI);
    }
    with (actor ACTOR_NPC_RURIRI) {
        Turn2DirectionLives(4, 10, ACTOR_NPC_MARIRU);
    }
    message_SetFace(ACTOR_NPC_MARIRU, FACE_SIGH, FACE_POS_BOTTOM_R_FACEINW);
    message_Talk({
        english=" It\'s too much for you. I don\'t\nwant you getting into any scary scrapes.",
    });
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_NPC_MARIRU) {
        Turn2DirectionLives(4, 10, ACTOR_ATTENDANT1);
    }
    WaitExecuteLives(ACTOR_NPC_MARIRU);
    with (actor ACTOR_NPC_RURIRI) {
        Turn2DirectionLives(4, 10, ACTOR_ATTENDANT1);
    }
    message_Talk({
        english=" I went to [CS:P]Amp Plains[CR] by myself.",
    });
    message_Talk({
        english=" But many Electric-type Pokémon\nlive there!",
    });
    message_Talk({
        english=" I\'m no match for them.",
    });
    message_SetFace(ACTOR_NPC_MARIRU, FACE_TEARY_EYED, FACE_POS_BOTTOM_R_FACEINW);
    message_Talk({
        english=" I tried going many times. But I\nget knocked out right away...",
    });
    message_Talk({
        english=" I just c-c-can\'t stand how weak\nI am...",
    });
    se_Play(8972);
    with (actor ACTOR_NPC_MARIRU) {
        SetEffect(EFFECT_SWEAT_DROPS_FROM_BOTH_SIDES_MEDIUM, 3);
    }
    with (actor ACTOR_NPC_MARIRU) {
        WaitEffect();
    }
    WaitExecuteLives(ACTOR_NPC_MARIRU);
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    bgm2_FadeOut(60);
    with (actor ACTOR_ATTENDANT1) {
        ExecuteCommon(CORO_JUMP_HAPPY_FUNC_SERIES, 0);
    }
    WaitExecuteLives(ACTOR_ATTENDANT1);
    message_SetFace(ACTOR_ATTENDANT1, FACE_NORMAL, FACE_POS_TOP_L_CENTER);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" No problem! I know what to do!",
            }
        case 2:
            {
                english=" No problem! I know what to do!",
            }
        default:
            {
                english=" No problem! I know what to do!",
            }
    }
    message_KeyWait();
    WaitBgm(BGM_GROWING_ANXIETY);
    bgm_PlayFadeIn(BGM_WIGGLYTUFFS_GUILD, 0, 256);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" We\'ll go and get the [CS:Z]Water Float[CR]\nback for you.",
            }
        case 2:
            {
                english=" We\'ll go get the [CS:Z]Water Float[CR]\nback for you.",
            }
        default:
            {
                english=" We\'ll go get the [CS:Z]Water Float[CR]\nback for you.",
            }
    }
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_NPC_RURIRI) {
        ExecuteCommon(CORO_JUMP_HAPPY_FUNC_SERIES, 0);
    }
    WaitExecuteLives(ACTOR_NPC_RURIRI);
    message_SetFace(ACTOR_NPC_RURIRI, FACE_NORMAL, FACE_POS_STANDARD);
    message_Talk({
        english=" Really?!",
    });
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_NPC_MARIRU) {
        ExecuteCommon(CORO_JUMP_ANGRY_FUNC_SERIES, 0);
    }
    WaitExecuteLives(ACTOR_NPC_MARIRU);
    message_SetFace(ACTOR_NPC_MARIRU, FACE_NORMAL, FACE_POS_BOTTOM_R_FACEINW);
    message_Talk({
        english=" You will?!",
    });
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    message_SetFace(ACTOR_ATTENDANT1, FACE_NORMAL, FACE_POS_TOP_L_CENTER);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" Yup![K] So no more crying!",
            }
        case 2:
            {
                english=" Yep![K] So no more tears!",
            }
        default:
            {
                english=" Yep![K] So no more tears!",
            }
    }
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_NPC_MARIRU) {
        ExecuteCommon(CORO_JUMP_HAPPY_FUNC_SERIES, 0);
    }
    WaitExecuteLives(ACTOR_NPC_MARIRU);
    message_SetFace(ACTOR_NPC_MARIRU, FACE_NORMAL, FACE_POS_BOTTOM_R_FACEINW);
    message_Talk({
        english=" OK![K] W-we\'re sorry to bother\nyou about this. Thank you!",
    });
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_ATTENDANT1) {
        SetEffect(EFFECT_LAUGHING, 3);
    }
    message_SetFace(ACTOR_ATTENDANT1, FACE_HAPPY, FACE_POS_TOP_L_CENTER);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" You should see your faces!\nAll that crying and laughing... C\'mon!",
            }
        case 2:
            {
                english=" You should see your faces!\nAll that crying and laughing... C\'mon!",
            }
        default:
            {
                english=" You should see your faces!\nAll that crying and laughing... C\'mon!",
            }
    }
    message_KeyWait();
    with (actor ACTOR_ATTENDANT1) {
        SetEffect(EFFECT_NONE, 3);
    }
    message_SetFace(ACTOR_ATTENDANT1, FACE_NORMAL, FACE_POS_TOP_L_CENTER);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" Don\'t worry, though.[K] We\'ll get it\nback for you. Sure thing!",
            }
        case 2:
            {
                english=" Don\'t worry, though.[K] We\'ll get it\nback for you. It\'s a promise!",
            }
        default:
            {
                english=" Don\'t worry, though.[K] We\'ll get it\nback for you. We promise!",
            }
    }
    CallCommon(CORO_MESSAGE_CLOSE_WAIT_FUNC);
    with (actor ACTOR_ATTENDANT1) {
        Turn2DirectionLives(4, 10, ACTOR_PLAYER);
    }
    WaitExecuteLives(ACTOR_ATTENDANT1);
    with (actor ACTOR_PLAYER) {
        Turn2DirectionLives(4, 10, ACTOR_ATTENDANT1);
    }
    WaitExecuteLives(ACTOR_PLAYER);
    message_SwitchTalk ($PARTNER_TALK_KIND) {
        case 1:
            {
                english=" Let\'s go, [hero]! Onward\nto [CS:P]Amp Plains[CR]!",
            }
        case 2:
            {
                english=" Let\'s go, [hero]! We\'re\noff to [CS:P]Amp Plains[CR]!",
            }
        default:
            {
                english=" Let\'s go, [hero]! Let\'s get\nto [CS:P]Amp Plains[CR]!",
            }
    }
    message_Close();
    with (actor ACTOR_PLAYER) {
        SetAnimation(71);
    }
    with (actor ACTOR_PLAYER) {
        WaitAnimation();
    }
    WaitExecuteLives(ACTOR_PLAYER);
    with (actor ACTOR_PLAYER) {
        SetAnimation(2);
    }
    Wait(15);
    with (actor ACTOR_NPC_MARIRU) {
        Turn2Direction(4, 10, DIR_LEFT);
    }
    with (actor ACTOR_NPC_RURIRI) {
        MovePositionMark(1, Position<'m2', 32, 28>);
    }
    with (actor ACTOR_NPC_RURIRI) {
        Turn2Direction(4, 2, DIR_LEFT);
    }
    Wait(15);
    dungeon_mode(17) = DMODE_OPEN;
    end;
}

