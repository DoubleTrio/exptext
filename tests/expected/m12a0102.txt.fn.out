58      [Azurill]:  [partner] and [hero]!\nHi there!
65      [Kunan](1):  Huh? It\'s you little guys...
69      [Kunan](2):  Oh! It\'s you little guys...
73      [Kunan](3):  Hey! It\'s you little guys...
79      [Kunan](1):  We heard we have visitors.\nAre you our visitors?
83      [Kunan](2):  We heard we have visitors.\nAre you our visitors?
87      [Kunan](3):  We heard we have visitors. Are\nyou maybe those visitors?
97      [Marill]:  Yep![K] We wanted to ask\nTeam [team:] for help.
108     [Kunan](1):  Help?
112     [Kunan](2):  Help?
116     [Kunan](3):  Help?
122     [Marill]:  Yep!
131     [Azurill]:  We need your help finding a\n[CS:Z]Water Float[CR].
150     [Kunan](1):  A [CS:Z]Water Float[CR]?
154     [Kunan](2):  A [CS:Z]Water Float[CR]?
158     [Kunan](3):  A [CS:Z]Water Float[CR]?
164     [Kunan](1):  You\'ve been searching for your\n[CS:Z]Water Float[CR], right?
168     [Kunan](2):  Weren\'t you looking for that\nearlier?[K] A [CS:Z]Water Float[CR]?
172     [Kunan](3):  Isn\'t that what you were looking\nfor earlier?[K] A [CS:Z]Water Float[CR]?
181     [Marill]:  Yep! Someone said one was lying\non the beach. We went there looking for it!
184     [Marill]:  We didn\'t find it. Instead, this is\nwhat we found.
221     [Kunan](1):  This scrap of paper?
225     [Kunan](2):  This scrap of paper?
229     [Kunan](3):  This scrap of paper?
235     [Kunan](1):  There\'s writing on it.
239     [Kunan](2):  There\'s writing on it.
243     [Kunan](3):  There\'s writing on it.
274     [message_Mail]: [CN][partner] read the message\n[CN]on the scrap of paper.
281     [Kunan](1):  Let\'s see. What does it say?[K]\n\"The [CS:Z]Water Float[CR] from the beach...\"
285     [Kunan](2):  Let\'s see. What does it say?[K]\n\"The [CS:Z]Water Float[CR] from the beach...\"
289     [Kunan](3):  Let\'s see. What does it say?[K]\n\"The [CS:Z]Water Float[CR] from the beach...\"
295     [Kunan](1):  \"...is now in our possession.\"
299     [Kunan](2):  \"...is now in our possession.\"
303     [Kunan](3):  \"...is now in our possession.\"
311     [Kunan](1):  \"Just try to take it from us!\nWe\'ll wait in the deepest part of [CS:P]Amp Plains[CR].\"
315     [Kunan](2):  \"Just try to take it from us!\nWe\'ll wait in the deepest part of [CS:P]Amp Plains[CR].\"
319     [Kunan](3):  \"Just try to take it from us!\nWe\'ll wait in the deepest part of [CS:P]Amp Plains[CR].\"
325     [Kunan](1):  \"But knowing how weak and puny\nyou are...\"
329     [Kunan](2):  \"But knowing how weak and puny\nyou are...\"
333     [Kunan](3):  \"But knowing how weak and puny\nyou are...\"
339     [Kunan](1):  \"...I bet you can\'t even reach us!\nChaw-haw-haw!\"
343     [Kunan](2):  \"...I bet you can\'t even reach us!\nChaw-haw-haw!\"
347     [Kunan](3):  \"...I bet you can\'t even reach us!\nChaw-haw-haw!\"
353     [Kunan](1):  \"Can\'t handle it? Go cry to your\nbig-shot friends! Chaw-haw-haw!\"[K]\nHey! This note! It\'s a...
357     [Kunan](2):  \"Can\'t handle it? Go cry to your\nbig-shot friends! Chaw-haw-haw!\"[K]\nHey! This note! It\'s a...
361     [Kunan](3):  \"Can\'t handle it? Go cry to your\nbig-shot friends! Chaw-haw-haw!\"[K]\nHey! This note! It\'s a...
377     [Kunan](1):  It\'s a ransom note! Isn\'t it?!
381     [Kunan](2):  It\'s a ransom note! Isn\'t it?!
385     [Kunan](3):  It\'s a ransom note! Isn\'t it?!
399     [Kunan](1):  You little guys shouldn\'t even\nthink about going!
403     [Kunan](2):  You little guys! Promise me you\nwon\'t go yourself.
407     [Kunan](3):  You little guys! Please don\'t even\nthink about going.
413     [Kunan](1):  It could be a trap.
417     [Kunan](2):  I suspect a trap.
421     [Kunan](3):  It\'s probably a trap!
428     [Marill]:  But the [CS:Z]Water Float[CR] is very\nimportant to us.
431     [Marill]:  We really want to get it back.
434     [Marill]:  But I can\'t take [CS:N]Azurill[CR] to a\ndangerous place.
444     [Azurill]:  I said I would go too!
455     [Marill]:  It\'s too much for you. I don\'t\nwant you getting into any scary scrapes.
466     [Marill]:  I went to [CS:P]Amp Plains[CR] by myself.
469     [Marill]:  But many Electric-type Pok√©mon\nlive there!
472     [Marill]:  I\'m no match for them.
476     [Marill]:  I tried going many times. But I\nget knocked out right away...
479     [Marill]:  I just c-c-can\'t stand how weak\nI am...
499     [Kunan](1):  No problem! I know what to do!
503     [Kunan](2):  No problem! I know what to do!
507     [Kunan](3):  No problem! I know what to do!
516     [Kunan](1):  We\'ll go and get the [CS:Z]Water Float[CR]\nback for you.
520     [Kunan](2):  We\'ll go get the [CS:Z]Water Float[CR]\nback for you.
524     [Kunan](3):  We\'ll go get the [CS:Z]Water Float[CR]\nback for you.
534     [Azurill]:  Really?!
543     [Marill]:  You will?!
550     [Kunan](1):  Yup![K] So no more crying!
554     [Kunan](2):  Yep![K] So no more tears!
558     [Kunan](3):  Yep![K] So no more tears!
568     [Marill]:  OK![K] W-we\'re sorry to bother\nyou about this. Thank you!
578     [Kunan](1):  You should see your faces!\nAll that crying and laughing... C\'mon!
582     [Kunan](2):  You should see your faces!\nAll that crying and laughing... C\'mon!
586     [Kunan](3):  You should see your faces!\nAll that crying and laughing... C\'mon!
597     [Kunan](1):  Don\'t worry, though.[K] We\'ll get it\nback for you. Sure thing!
601     [Kunan](2):  Don\'t worry, though.[K] We\'ll get it\nback for you. It\'s a promise!
605     [Kunan](3):  Don\'t worry, though.[K] We\'ll get it\nback for you. We promise!
620     [Kunan](1):  Let\'s go, [hero]! Onward\nto [CS:P]Amp Plains[CR]!
624     [Kunan](2):  Let\'s go, [hero]! We\'re\noff to [CS:P]Amp Plains[CR]!
628     [Kunan](3):  Let\'s go, [hero]! Let\'s get\nto [CS:P]Amp Plains[CR]!
